elixir/apps/web/lib/web/live/sites/new_token.ex

```diff
@@ -236,20 +257,29 @@ defmodule Web.Sites.NewToken do
             echo "Unsupported architecture"; \\
             exit 1 ;; \\
         esac; \\
-        wget -O /usr/local/bin/firezone-gateway $bin_url; \\
-        chmod +x /usr/local/bin/firezone-gateway; \\
-        mkdir -p /etc/firezone; \\
-        chmod 0755 /etc/firezone; \\
-        iptables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        iptables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        iptables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
-        ip6tables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
-        ip6tables-nft -A FORWARD -o tun-firezone -j ACCEPT; \\
-        ip6tables-nft -t nat -A POSTROUTING -o e+ -j MASQUERADE; \\
+        curl -Ls $bin_url -o /usr/local/bin/firezone-gateway; \\
+        chgrp firezone /usr/local/bin/firezone-gateway; \\
+        chmod 0750 /usr/local/bin/firezone-gateway; \\
+        setcap 'cap_net_admin+eip' /usr/local/bin/firezone-gateway; \\
       fi; \\
+      groupadd -f firezone; \\
+      id -u firezone &>/dev/null || useradd -r -g firezone -s /sbin/nologin firezone; \\
+      mkdir -p /var/lib/firezone; \\
+      chown firezone:firezone /var/lib/firezone; \\
+      chmod 0775 /var/lib/firezone; \\
+      iptables-nft -A FORWARD -i tun-firezone -j ACCEPT; \\
```

Comment:
This issue here is that it won't just run on every restart of the system but also will flood the iptables rules if the gateway crash loops.

Reply:
Hmm yeah it didn't seem like it would add dupe rules when I tested but I'll make sure this won't happen

Reply:
Fixed by checking for rule existence with `-C` first.